packages/devextreme/js/__internal/grids/grid_core/column_fixing/m_column_fixing.ts

```diff
@@ -1148,6 +1149,18 @@ const resizing = (Base: ModuleType<ResizingController>) => class ResizingColumnF
   }
 };
 
+const keyboardNavigation = (Base: ModuleType<KeyboardNavigationController>) => class KeyboardNavigationExtender extends Base {
+  protected _toggleInertAttr(value = false): void {
```

Comment:
Let's remove the default argument value
Because we don't use this method without passing the value + passing the value is better than the default one defined in the method (more obvious)

```suggestion
  protected _toggleInertAttr(value: boolean): void {
```